index.js

```diff
@@ -0,0 +1,39 @@
+function checkLowerCase(currentLetter) {
+  return currentLetter >= 97 && currentLetter <= 122
+}
+
+function passwordLength(password) {
+  if (password.length < 8) {
+    return false
+  }
+
+  return true
+}
+
+function validatePassword(password) {
+  let length = passwordLength(password)
+  let lowerCaseCount = 0
+
+
+  for (let i = 0; i < password.length; i++) {
+    let currentLetter = password[i]
+
+    if (checkLowerCase(currentLetter)) {
+      lowerCaseCount++
+    }
+
+    return checkLowerCase(password) && passwordLength(password)
```

Comment:
why check lower case each time if it's already been possibly set to more than 0 already in lowerCaseCount?

also if at this point the password has a lower case and a 8 length. it will return true... skiping check for upper case and special chars.

Reply:
why pass in whole password to check lower case? instead of character.